distribution/src/main/resources/docker-compose/docker-compose-minimal.yml

```diff
@@ -106,6 +118,7 @@ services:
       JAVA_OPTS: " -Xms256m -Xmx3000m"
       ACTIVEMQ_URL: "nio://activemq:61616"
       FILE_STORE_URL: "http://shared-file-store:8099/alfresco/api/-default-/private/sfs/versions/1/file"
+      LOGGING_LEVEL_ORG_ALFRESCO_TRANSFORM_BASE_TRANSFORM: TRACE
```

Comment:
Similarly is this going to flood the logs?

Reply:
Same here - this logs are very helpful in problem resolving

Reply:
Regarding if it will flood the logs - most of the logs will be generated while `bulk-ingester` will run (similarly like in case of 'live-ingester` and it's logs). There are two log entries per one T-request in each container

Reply:
Ok - sounds reasonable